src/frontend/src/flows/faq/index.ts

```diff
@@ -1,10 +1,107 @@
-export { questions } from "./questions";
+/** All the questions displayed on the FAQ */
+export const questions = {
+  windowsHello: {
+    priority: 10,
+    question: "Does Internet Identity support Windows Hello?",
+    anchor: "windows-hello",
+    answer: html`<span
+      >Yes! Internet Identity supports authenticating via
+      ${mkExternalLink({
+        href: "https://support.microsoft.com/en-us/windows/learn-about-windows-hello-and-set-it-up-dae28983-8242-bb2a-d3d1-87c9d265a5f0",
+        text: "Windows Hello",
+      })}.
+      If Windows Hello is set up on your PC then Internet Identity will offer
+      you to authenticate through Windows Hello.</span
+    >`,
+    links: [
+      {
+        name: "Windows Hello Guide for Internet Identity",
+        link: "https://sdk.dfinity.org/docs/ic-identity-guide/hello-guide.html",
+      },
+    ],
+  },
+  lostDevice: {
+    priority: 1,
+    question: "If I lose my device, can I still use Internet Identity?",
+    anchor: "lost-device",
+    answer:
+      "If you have an Identity Anchor tied to only one device and you lose that one device, you will be locked out. As a best practice, we recommend adding multiple devices multiple devices and recovery mechanisms to every Identity Anchor.",
+    links: [
+      {
+        name: "Device Management How-To",
+        link: "https://sdk.dfinity.org/docs/ic-identity-guide/auth-how-to.html",
+      },
+    ],
+  },
+  moreDevices: {
+    priority: 1,
+    question: "How do I add more devices to my Identity Anchor?",
+    anchor: "more-devices",
+    answer:
+      "To add more devices to an existing Identity Anchor, please see the guide here:",
+    links: [
+      {
+        name: "How to add a device",
+        link: "https://sdk.dfinity.org/docs/ic-identity-guide/auth-how-to.html#_add_a_device",
+      },
+    ],
+  },
+  shareIIAnchor: {
+    priority: 10,
+    question:
+      "Does Internet Identity share my anchor or personal information with DApps?",
+    anchor: "share-ii-anchor",
+    answer:
+      'No. Internet Identity uses a different Principal (a "pseudonym") for each DApp that you authenticate to using Internet Identity. Since the pseudonyms Internet Identity generates for you are different for each DApp, DApps cannot use them to track you outside of their realm.',
+    links: [],
+  },
+  invalidSeedphrase: {
+    priority: 10,
+    question:
+      'Why do I get "Invalid Seedphrase" when I try to recover my Identity Anchor?',
+    anchor: "invalid-seedphrase",
+    answer: `
+    Most likely, it means that the recovery phrase (also known as seed phrase) is not BIP39 compatible.
+
+    A recovery phrase generated by Internet Identity will contain an anchor number followed by some words. For instance, "135033-squirrel-soccer-...". The number is the anchor for which the recovery phrase is generated; note that Internet Identity will strip the number before trying to recover an Identity, meaning the number is only meant for humans!
```

Comment:
>135033-squirrel-soccer-

NIT: Not changed in this PR, but these should be spaces.

Reply:
Thanks! #636